Makefile

```diff
@@ -50,11 +51,8 @@ repl:
 	dune utop src/haz3lcore
 
 test:
-	dune build @src/fmt --auto-promote || true
-	dune exec src/hazeltest/hazeltest.exe -- --regression-dir src/hazeltest/regressions
-
-reset-regression-tests:
-	dune exec src/hazeltest/hazeltest.exe -- regression --regression-dir src/hazeltest/regressions --reset-regressions
+	dune build @src/fmt --auto-promote src --profile dev || true
+	node $(TEST_DIR)/haz3ltest.bc.js test
```

Comment:
We should add instructions on installing `node` to the install instructions since it is now a dependency.